packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -30,7 +30,7 @@ export class DeltaScheduler {
 	public static readonly processingTime = 50;
 
 	// The increase in time for processing ops after each turn.
-	private readonly processingTimeIncrement = 10;
+	public static processingTimeIncrement = 10;
```

Comment:
This should not be changed, right?
```suggestion
	public static readonly processingTimeIncrement = 10;
```